WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -214,9 +218,12 @@ class PlanDetailViewController: UIViewController {
                 }
                 
                 return ImmuTable(sections: groups.map { group in
-                    let features = group.slugs.map { PlanService.featureForPlan(plan, withSlug: $0) }
-                    
-                    return ImmuTableSection(headerText: group.title, rows: features.map({ TextRow(title: $0!.title, value: $0!.description) }), footerText: nil)
+                    let features = group.slugs.flatMap { PlanService.featureForPlan(plan, withSlug: $0) }
+                    let rows: [ImmuTableRow] = features.map({ feature in
+                        let iconName = "features-\(feature.slug).png"
```

Comment:
I'm guessing this will eventually go away, but maybe make it a `PlanFeature` property in the meantime like we do with `Plan`?